kvdb/sqlite/config.go

```diff
@@ -0,0 +1,13 @@
+package sqlite
+
+import "time"
+
+// Config holds sqlite configuration data.
+//
+//nolint:lll
+type Config struct {
+	DBPath         string        `long:"dbpath" description:"The path to the sqlite database file."`
+	Timeout        time.Duration `long:"timeout" description:"The time after which a database query should be timed out."`
+	BusyTimeout    time.Duration `long:"busytimeout" description:"The maximum amount of time to wait for a database connection to become available for a query."`
+	MaxConnections int           `long:"maxconnections" description:"The maximum number of open connections to the database. Set to zero for unlimited."`
```

Comment:
Non blocking, but it could be cool to add a raw pragma string here. Then ppl will be able to tune w/e values we don't expose directly on their own w/o recompiling.